### ISSUE TITLE
Specify merging type from topic branch to master

### ISSUE THREAD

**Issue by Mindexperiment** [rocketraman/gitworkflow#10]
Hi all,

I found that there's a bit of confusion when you document the merging of a topic branch inside master, there is no description on what type of merging is used (fast-forward or --no-ff).

By looking to the visual image it seems that's a --no-ff (hope it is) merge but it's never clearly mentioned in the description.

Thank you guys

**Comment by rocketraman** [comment]
Yes, that's correct. Merging a topic to `master` is basically the same as to `pu` or `next` -- its no different really than any other "integration" branch except it is long-lived and does not get rewound and rebuilt from time to time.

There are several TODOs in the docs related to this. I'll get around to it eventually, or feel free to submit a pull request!
